data/human/intro missions.txt

```diff
@@ -423,14 +516,14 @@ mission "Intro [3 Interceptor]"
 	description "Drop James off on <destination>, where he has a spot reserved in a retirement home."
 	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I found you another mission, but it looks like you'll need <capacity> in order to take it on. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
 	passengers 1
-	source "Ingot"
+	source "New Sahara"
 	destination "Hestia"
 	to offer
-		has "Intro [2 Interceptor]: done"
+		has "Intro [2.1 Interceptor]: done"
 	
 	on offer
 		conversation
-			`You find James sitting at the spaceport's tiny bar, sipping a beer while listening to the steady patter of micrometeorites against the roof. "Working here," says James, "is one of the few jobs in the galaxy more dangerous than fighting pirates. And worse, because if a big meteorite decides to strike this station, there's basically nothing they can do about it except hope they see it coming. Care for a drink?"`
+			`You find James sitting at the spaceport's tiny bar, sipping a beer while watching the large displays that show the landing pads and the irrigated fields beyond. The displays are suppposed to make the underground building feel less claustrophobic, but living on a ship, that doesn't bother you much. "Care for a drink?" he asks.`
```

Comment:
```suggestion
			`You find James sitting at the spaceport's tiny bar, sipping a beer while watching the large displays that show the landing pads and the irrigated fields beyond. The displays are supposed to make the underground building feel less claustrophobic, but living on a ship, that doesn't bother you much. "Care for a drink?" he asks.`
```

Reply:
Also, you've only lived on the ship for something like a week? Maybe remove that line?